packages/itwinui-react/src/core/Checkbox/Checkbox.tsx

```diff
@@ -116,9 +127,10 @@ export const Checkbox = React.forwardRef((props, ref) => {
           [`iui-${status}`]: !!status,
           'iui-loading': isLoading,
         },
-        className,
+        wrapperClassName,
```

Comment:
Now that the class name for the label comes from `wrapperClassName`, there is a check in the `checkbox` element that only adds `className` to the checkbox when there is no label. This check can be removed and `className` can be added directly to the checkbox element regardless of if there is a label or not

Reply:
`className` is still needed for users to add a custom class name to `iui-checkbox`. Only the check for `label` needed to be removed